storage/mongodb/mongodb.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+vars_file='external_vars.yaml'
+output_file="mongodb.out"
+rm $output_file
+mongodb_version=$(cat $vars_file | grep MONGODB_VERSION | cut -d ' ' -f 2)
+
+if [[ $mongodb_version == "latest" ]]; then
```

Comment:
ohhh... I like this!

Reply:
I would add information in README you can use the latest version - and maybe change that in external_vars file - we want to run against the latest one - if that is not some retest.

Reply:
Updated readme to explain the latest variable in the external_vars. And pointed to run mongodb.sh instead of runmongo.sh